backend/app/routes/contactUs/index.js

```diff
@@ -7,7 +7,7 @@ const deleteContactUs = require('./delete');
 const { authMiddleware } = require('../../../helpers/middlewares/auth');
 
 router.get('/getcontactus', getContact);
```

Comment:
can u add the authMiddleware here as only admins should be able to fetch this data